Sources/PointFree/Account/Invite.swift

```diff
@@ -122,6 +122,32 @@ let acceptInviteMiddleware: Middleware<StatusLineOpen, ResponseEnded, Tuple2<Tea
         .flatMap(const(conn |> redirect(to: path(to: .account(.index)))))
 }
 
+let addTeammateViaInviteMiddleware: Middleware<
+  StatusLineOpen,
+  ResponseEnded,
+  Tuple2<User?, EmailAddress?>,
+  Data
+  > =
+  filterMap(require1 >>> pure, or: loginAndRedirect)
+    <<< filterMap(require2 >>> pure, or: invalidSubscriptionErrorMiddleware)
+    <<< requireStripeSubscription
+    <<< requireActiveSubscription
+    <| { (conn: Conn<StatusLineOpen, Tuple3<Stripe.Subscription, User, EmailAddress>>) in
+
+      let (stripeSubscription, inviter, email) = lower(conn.data)
+      let newPricing = Pricing(
+        billing: stripeSubscription.plan.interval == .month ? .monthly : .yearly,
+        quantity: stripeSubscription.quantity + 1
+      )
```

Comment:
are the above lines of code a legit way to compute the new pricing? we should maybe combine `Plan.Interval` and `Pricing.Billing` into a single enum

Reply:
this should be legit for how we structure plans right now